### PR TITLE
Replaced Create React App AMD example with a Webpack AMD example

### PR DESCRIPTION
**Pull request by maxburs**
Create React App isn't very actively maintained, and pulls in a lot of dependencies. I've replaced our old, complex, and not working, CRA example with a smaller Webpack example.

Webpack code was created by calling [webpack init](https://webpack.js.org/api/cli/#init).

### CODE REVIEW

File: samples/amd-webpack-react/src/index.js [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+    },
+};
+
+function App() {
+    const divRef = React.useRef(null);
+
+    React.useLayoutEffect(() => {
+        let disposed = false;
+        let editor;
+        // https://webpack.js.org/api/module-variables/#__non_webpack_require__-webpack-specific
```
Comment (Weffe) [inline samples/amd-webpack-react/src/index.js:41]: I'd maybe add a comment here why we need this special `__non_webpack_require__()`? Or if in the context of an AMD module that this makes sense inherently then this link is enough.

File: samples/amd-webpack-react/package.json [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    "version": "1.0.0",
+    "private": true,
+    "scripts": {
+        "start": "webpack watch",
+        "build": "webpack --mode=production --node-env=production",
+        "build:dev": "webpack --mode=development",
+        "build:prod": "webpack --mode=production --node-env=production",
+        "watch": "webpack --watch",
+        "serve": "webpack serve",
+        "copy-amd-files": "copyMonacoFilesAMD ./public"
```
Comment (Weffe) [inline samples/amd-webpack-react/package.json:12]:
```suggestion
```

Can we delete?

Comment (maxburs) [reply samples/amd-webpack-react/package.json:12]: No, this script must be run before the example works. See the README.md

**Review by Weffe — APPROVED** [review]
LGTM

**Pull request merged**